DTS-STN/canadian-dental-care-plan
# Demographic equity type codes now mocked.

### Description


.env.example updated to include relevant cache duration and type ID Mocked list of equity types codes added.
Handler handed lookup api server
demographics-part1 route updated
Relevant schema added.
New message properties added.

### Related Azure Boards Work Items

[AB#2990](https://dev.azure.com/DTS-STN/Canada%20Dental%20Care%20Plan/_workitems/edit/2990)

### Screenshots (if applicable)


![Screenshot from 2024-03-08 07-22-56](https://github.com/DTS-STN/canadian-dental-care-plan/assets/154452024/598e0337-9892-4492-85cb-bd8f787fd085)


### Checklist


### Test Instructions
1. Open the application.
2. Click one of the language buttons.
3. Click accept.
4. Specify that you are applying for yourself.
5. Manually edit the URL to replace 'tax-filing' with 'demographic-part1'
6. Observe that the new question for equity types is shown
7. Select the 'Other answer' option from the list of demographic types
8. Observe that Please specify text box is now showing
